### ISSUE TITLE
React versioning bug

### ISSUE THREAD

**Issue by sarahzinger** [casesandberg/react-color#264]
I currently have React@15.1.0 installed and react-color@2.3.3 has installed react@15.3.2 despite listing react as a peer dependency. I then receive the following error in the console:

```
Uncaught Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://fb.me/react-refs-must-have-owner).
```

If I then update my app to use react@15.3.2 the error goes away and the color picker works great (and relies on the peer dependency rather than installing it's own version of react). Ideally react-color would rely on the peer dependency without the upgrade.

**Comment by casesandberg** [comment]
That is really bizarre, it is in peerDependencies so it shouldn't be an issue. I will look into it :)

**Comment by sarahzinger** [comment]
Thanks! It does look like it should work as expected by my understanding of semver is not stellar. I wonder if it's a problem with npm?

**Comment by mnzt** [comment]
I was having this issue, I removed
```
    externals: {
        'react': 'React'
    },
```

from my `webpack.config.js` file and the issue has subsided.

**Comment by sarahzinger** [comment]
I'm not using webpack? Any suggestions? Thanks!

**Comment by sarahzinger** [comment]
This bug has happened again as React@14.0.0 was released on wed but my app was relying on react@15.3.2 now. Is there a recommended long term strategy for npm2 users to deal with this issue? Thanks!
